CODE-OF-CONDUCT.md

```diff
@@ -34,6 +34,34 @@ In keeping with this commitment, we offer the following guidelines:
      in [Security Reviews:
      Outcome][review-outcome]
 
+
+# Incident handling and escalation
+
+Content for the purposes of the code of conduct as well as incident is defined not only as published or draft content but also online discourse, such as slack messages or emails, and interactions at in-person events
+If an incident involving community conduct occurs, please follow the guidelines below on how to handle and report the issue:
+
+- If you see content that clearly does not meet the official Code of Conduct, please send an e-mail to the Co-Chair/TL mailing list (cncf-tag-security-leads@lists.cncf.io) and the creator of the content. (For more details refer to the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)).
+- If you are uncomfortable with a piece of content (but it may not necessarily violate the COC), we suggest sending a private message to the content owner expressing your concerns. If this is not resolved, you may wish to request the help of a Co-Chair/TL via cncf-tag-security-leads@lists.cncf.io to help mediate the situation.
```

Comment:
```suggestion
- If you are uncomfortable with a piece of content (but it may not necessarily violate the code of conduct), we suggest sending a private message to the content owner expressing your concerns. If this is not resolved, you may wish to request the help of a Co-Chair/TL via cncf-tag-security-leads@lists.cncf.io to help mediate the situation.
```